app/views/root/_google_analytics.html.erb

```diff
@@ -14,9 +14,11 @@
   if (window.devicePixelRatio) {
     _gaq.push(['_setCustomVar', 11, 'Pixel Ratio', String(window.devicePixelRatio), 2 ]);
   }
-  // Track status code for failed responses
+  // Track status code
   if (window.httpStatusCode) {
     _gaq.push(['_setCustomVar', 15, 'httpStatusCode', String(window.httpStatusCode), 3 ]);
+  } else {
+    _gaq.push(['_setCustomVar', 15, 'httpStatusCode', 'unset', 3 ]);
```

Comment:
Rather than having making a different call in the conditional it would be safer to being something like:

```javascript
if (!window.httpStatusCode) {
  window.httpStatusCode = 'unset';
}
_gaq.push(['_setCustomVar', 15, 'httpStatusCode', String(window.httpStatusCode), 3 ]);
```

That would reduce the chance of a developer accidentally only updating one of the setCustomVar calls. If for example we wanted to use a different custom variable number.